PWGHF/TableProducer/candidateCreatorB0.cxx

```diff
@@ -79,6 +80,13 @@ struct HfCandidateCreatorB0 {
   Filter filterSelectTracks = (!usePionIsGlobalTrackWoDCA) || ((usePionIsGlobalTrackWoDCA) && requireGlobalTrackWoDCAInFilter());
   Filter filterSelectCandidates = (aod::hf_sel_candidate_dplus::isSelDplusToPiKPi >= selectionFlagD);
 
+  using TracksWithSelFiltered = soa::Filtered<TracksWithSel>;
```

Comment:
unfortunately we cannot filter them because otherwise the lookup of the track index (see `auto trackPion = trackId.track_as<TracksWithSelFiltered>();`) might fail if the track has been filtered out